### PR TITLE
Update to crate-docs-theme 0.5.70

### PR DESCRIPTION
**Pull request by MarkusH**

### CODE REVIEW

**Comment by MarkusH** [comment]
See passing docs build in https://readthedocs.org/projects/croud/builds/8521702/

File: .gitignore [hunk 1]
```diff
@@ -58,7 +58,7 @@ coverage.xml
 *.log
 
 # Sphinx documentation
-docs/_build/
+docs/docs/_build/
```
Comment (mechanomi) [inline .gitignore]: this isn't right? how is that dir being generated

Comment (MarkusH) [reply .gitignore]: The .gitignore is in the root directory. the docs are in `docs/docs/_build`:
```
$ tree
.
├── CHANGES.rst
├── CONTRIBUTING.rst
├── croud
│   ├── ...
│   ├── cmd.py
│   ├── ...
│   └── util.py
├── ...
├── docs
│   ├── bin
│   │   └── sphinx
│   ├── bootstrap.sh
│   ├── docs
│   │   ├── _build
│   │   │   ├── commands.html
│   │   │   ├── ...
│   │   │   └── _static
│   │   │       ├── ajax-loader.gif
│   │   │       ├── ...
│   │   │       └── websupport.js
│   │   ├── commands.txt
│   │   ├── conf.py
│   │   ├── getting-started.txt
│   │   └── index.txt
└── tox.ini
```

Comment (mechanomi) [reply .gitignore]: I see. this is a bad file system layout. I will create an imped to fix it

but your PR is good

**Pull request merged**